src/app/shared/services/posts-fake.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { Post } from '../interfaces/post.interface';
+
+const FAKE_POSTS: Post[] = [
+  { author: 'John Dow', title: 'Natural language interface accessibility', date: new Date().toDateString(), likeNumber: 20, content: 'lorem' },
+  { author: 'Anderson Bred', title: 'Accessibility of Remote Mettings', date: new Date().toDateString(), likeNumber: 2, content: 'lorem' },
+]
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PostsFakeService {
```

Comment:
let's use inheritance. Fake service should extend native service and mock some methods